model/import/Parser/TemplateDataParser.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\model\import\Parser;
+
+use oat\oatbox\service\ConfigurableService;
+
+class TemplateDataParser extends ConfigurableService
+{
+    public function parser(array $finalHeader): array
```

Comment:
Some points:

- We actually do not need the `$finalHeader`.
- Would it be possible to get the values base on a `TemplateInterface` instead?
- This class could only return the lines (not the header).
- Could we call it something more explicit, like: `CsvTeamplateSampleLineFactory`. Cause it seems we create the sample lines based on some template.

Reply:
It was done